doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_external_memory.rst

```diff
@@ -122,18 +120,46 @@ To enable the external memory, you must apply the :file:`nrf/samples/suit/smp_tr
 
       The companion image can be optionally upgraded and have its integrity checked.
 
-#. Enable the :kconfig:option:`CONFIG_SUIT_EXTERNAL_MEMORY_SUPPORT` Kconfig option, which enables the build of the reference companion image to be used as a child image of the application firmware.
-   It also enables other additional options that are required for the external memory DFU to work.
+#. Build and flash the application by completing the following:
+
+   .. code-block:: console
+
+      $ west build -b nrf54h20dk/nrf54h20/cpuapp --sysbuild
+      $ west flash
+
+   The build system will automatically generate a :file:`build/suit.zip` archive, which contains the SUIT envelope and candidate images.
```

Comment:
I am not sure if the suit.zip file will be ready for this release, @e-rk  can we say something about this?

Reply:
If it turns out that the mobile app DFU is not ready for release, I'll remove this section.